dvc/cli.py

```diff
@@ -102,11 +139,47 @@ def _find_parser(parser, cmd_cls):
 class DvcParser(argparse.ArgumentParser):
     """Custom parser class for dvc CLI."""
 
+    cmd_choices = []
+    sub_cmd_choices = {}
```

Comment:
These can probably be combined into a single dict (`sub_cmd_choices.keys()` is equivalent to `cmd_choices`). For commands with no subcommands, they can just be mapped to `None` or an empty list.